react/components/Button/README.md

```diff
@@ -157,6 +167,16 @@ Box types
 </div>
 ```
 
+Link mode
+
+```js
+<div>
+  <div className="mb4">
+    <Button variation="primary" href="http://vtex.com" target="_blank">Sign in</Button>
```

Comment:
Not sure this makes sense. A link with a Button's face is a UX bug imo. Also, we have a [Link](https://github.com/vtex/styleguide/pull/670) component on the way.

Reply:
no, there are various cases in which a link has the semantics of a button. ("Sign up", "Create new account", "Delete" buttons that go to a confirmation page, etc)